### DISCUSSION TITLE
Feature Request / Reports

### DISCUSSION THREAD

**Discussion by Sineos** [phoddie/node-red-mcu#94] [Q&A] [answered]
More a general thing:
For feature requests or reports where NR-MCU behaves differently to PC-NR: Do you prefer a discussion or an FR Issue?

**Comment by phoddie** [comment, marked as answer]
There will always be differences between Node-RED running on a computer and running on a microcontroller. Some are simply bugs, some are intentional implementation limits, and some are unresolvable (for example, a computer has infinite memory relative to a microcontroller). Probably best to start with discussion thread. GitHub allows transferring a Discussion to an Issue if it turns out to be a bug to be addressed.
